PyPortal_Wakeup_Light/wake_up_light.py

```diff
@@ -83,6 +84,12 @@
 pyportal.splash.append(wakeup_time_textarea)
 pyportal.splash.append(light_on_time_textarea)
 
+try:
```

Comment:
put this in a loop

Reply:
Oh right. What kind of loop should I put it in? It's usually just in the `while True:` loop but I had to call `pyportal.get_local_time()` outside of the `while True:` loop  one time before populating an array with wake up time objects. Otherwise the wake up time objects were not right. Is there a way to call pyportal.get_local_time() without using a try/except block?

Reply:
visit me